include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -256,6 +308,20 @@ class ADIOS2IOHandlerImpl
                                          std::string const & var );
 }; // ADIOS2IOHandlerImpl
 
+/*
+ * The following strings are used during parsing of the JSON configuration
+ * string for the ADIOS2 backend.
+ * Note that this struct's members additionally need to be defined at namespace
+ * scope up until C++17, see ADIOS2IOHandlerImpl.cpp
+ */
+struct ADIOS2Defaults
```

Comment:
Is there an advantage by doing this in a struct with `static` members over just taking a `namespace` with the same name and avoiding all the `static` handling?

Reply:
Yeah, using a namespace is better. Fixed it.